tools/vagrant/README.md

```diff
@@ -176,46 +186,46 @@ cd ${OPENWHISK_HOME}
 ./gradlew distDocker
 ```
 
-## Safe Re-deploy (after VM restart)
+## Using docker-runc
+Only for experimental use:
+To use docker-runc the docker-runc CLI use in the invoker needs to match the version on the docker engine host.
+Get the version of the docker engine like the following:
```

Comment:
is this to find the version of the `docker-runc` CLI used for deploying openwhisk docker images or the one used by the invoker?  i'm thinking this is the former.. probably should say so for clarity